server.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config();
+const source = require('rfr');
+
+const app = source('app');
+const Bot = source('lib/bot');
```

Comment:
Im not all that familiar with how some javascript requires work, but is `Bot` a singleton  does it need to be if we are `require()`ing it in multiple places and they should all be the same bot that is being used.

Reply:
I believe it works because we just get the client id, which is a static value. However, i realized there wasnt really a point to do it this way, and just hardcoded the url

Reply:
The point of it is so that the bot still works properly for local tests, plus hardcoding values is not good

Reply:
It still will work properly for local tests, it just displays the invite to server for Botomir as opposed to a testing bot. While i agree that hardcoding values is not good, the value should be the same every time our user interacts with it. We could make it an environmental variable instead?

Reply:
It works for botomir, but not if someone else wants to copy and paste the server. Imo it should be left as it was. 
Having an environment variable makes less sense to me because it is a value that is inherently unique to the bot user that the bot is running as, and since it can be pulled from the bot, that is the most appropriate place to do that. 

I'm pretty strongly against hardcoded any sort of configuration value into the codebase.

Reply:
I can change it back no problem, but are you okay with requiring bot in multiple places?

Reply:
Its worked so far so  ¯\_(ツ)_/¯